### DISCUSSION TITLE
Building Theia on a build Server with no internet egress

### DISCUSSION THREAD

**Discussion by arunswarnam** [eclipse-theia/theia#10558] [Q&A]
Hi! I am trying to build Theia on a machine that doesn't have access to internet and facing this issue during post install of one of the Theia's indirect dependencies: https://github.com/microsoft/vscode-ripgrep/issues/27

```
verbose 254.0814148 Error: /root/bld/input_build/node_modules/vscode-ripgrep: Command failed.
Exit code: 1
Command: node ./lib/postinstall.js
Arguments: 
Directory: /root/bld/input_build/node_modules/vscode-ripgrep
Output:
Finding release for v13.0.0-2
GET https://api.github.com/repos/microsoft/ripgrep-prebuilt/releases/tags/v13.0.0-2
events.js:200
      throw er; // Unhandled 'error' event
      ^

Error: connect ECONNREFUSED 
```
Is there a workaround to avoid accessing https://api.github.com post install?

**Comment by vince-fugnitto** [comment]
@arunswarnam thank you for the discussion, please see https://community.theia-ide.org/t/error-building-theia-behind-nexus/2136/2.

**Reply by arunswarnam** [reply]
Thank you @vince-fugnitto
